tasks/main.yml

```diff
@@ -16,9 +16,17 @@
        | selectattr('state', 'equalto', 'absent') | list) != libvirt_vms
 
 # Libvirt requires qemu-img to create qcow2 files.
-- name: Ensure qemu-img is installed
+- name: Ensure qemu-img and ovmf is installed
+  vars:
+    is_efi: >-
+      (libvirt_vms | selectattr('state', 'defined')
+        | selectattr('state', 'equalto', 'absent') |
+        | selectattr('boot_firmware', 'equalto', 'efi')) | bool
```

Comment:
Will this fail if boot_firmware is not defined for a VM? Based on L58 it should be optional.

Reply:
It hasn't failed in Gerrit change - https://review.opendev.org/c/openstack/kayobe/+/921660

Reply:
That's because that change sets boot_firmware :)

Reply:
Right, a lot of | default(something) ;-)